src/styles/antisnake.scss

```
@@ -0,0 +1,21 @@
+@import "_variables.scss";
+
+.antisnake {
+  display: grid;
+  width: 100%;
+  height: 100vh;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-template-rows: repeat(auto-fill);
+  @for $index from 1 through $block-sum {
+    & > div:nth-child(#{$index}) {

```

Зайва специфічність + не можна використовувати селектори по елементу
```suggestion
    & :nth-child(#{$index}) {

```